### PR TITLE
Fixes #29201 - Non-admin users are unable to view recurring logics

### PR DESCRIPTION
**Pull request by ranjan**

### CODE REVIEW

**Comment by ranjan** [comment]
> Works well, but rubocop is not happy.
@adamruzicka, I am unable to run rubocop locally, it fails with error
```
#rubocop app/controllers/foreman_tasks/recurring_logics_controller.rb 

.rubocop_todo.yml: Metrics/LineLength has the wrong namespace - should be Layout
Error: The `Layout/TrailingBlankLines` cop has been renamed to `Layout/TrailingEmptyLines`.
(obsolete configuration found in .rubocop_todo.yml, please update it)
The `Naming/UncommunicativeMethodParamName` cop has been renamed to `Naming/MethodParameterName`.
(obsolete configuration found in .rubocop_todo.yml, please update it)
``` 
Let me know what is the change required to make rubocop happy?

File: app/controllers/foreman_tasks/recurring_logics_controller.rb [hunk 1]
```diff
@@ -49,5 +49,10 @@ def filter(scope)
       scope.search_for(params[:search])
            .paginate(:page => params[:page], :per_page => params[:per_page])
     end
+
+    def controller_permission
+      'recurring_logics'
+    end
+
```
Comment (adamruzicka) [inline app/controllers/foreman_tasks/recurring_logics_controller.rb]: Rubocop complains about this line
```
app/controllers/foreman_tasks/recurring_logics_controller.rb:56:1: C: Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body end.
```

**Comment by adamruzicka** [comment]
@ranjan could you rebase please?

**Comment by ranjan** [comment]
> @ranjan could you rebase please?

Done

**Comment by adamruzicka** [comment]
Thank you @ranjan & @pondrejk !

**Pull request merged**